pkg/sql/crdb_internal.go

```diff
@@ -1131,6 +1134,40 @@ CREATE TABLE crdb_internal.session_trace (
 	},
 }
 
+// crdbInternalInflightTracingTable exposes the node-local registry of in-flight spans.
+var crdbInternalInflightTracingTable = virtualSchemaTable{
+	comment: `in-flight spans (RAM; local node only)`,
+	schema: `
+CREATE TABLE crdb_internal.node_inflight_tracing (
```

Comment:
Thoughts on the vtable name appreciated! I wonder if it would be more useful to call it `node_inflight_spans` considering that is what people will be able to access. but the session tracing vtable is called `session_trace`. (I could also make this one `node_inflight_trace` to be even more consistent.